cloudbridge/providers/gcp/provider.py

```diff
@@ -12,9 +12,15 @@
 import googleapiclient
 from googleapiclient import discovery
 
+import google_auth_httplib2
+import httplib2
+
 from oauth2client.client import GoogleCredentials
 from oauth2client.service_account import ServiceAccountCredentials
 
+from google.auth.credentials import with_scopes_if_required
+from google.oauth2.service_account import Credentials
```

Comment:
```suggestion
```

Reply:
This Credentials module is needed below (on line 347) so why remove it?

Reply:
This was an artefact from the previous review. Looks good to me thanks